### ISSUE TITLE
tview on a pi3hat with multiple servos does not work reliably

### ISSUE THREAD

**Issue by jpieper** [mjbots/moteus#16]
Running tview on a pi3hat with multiple servos often results in desynchronization of the command stream.  Power cycling sometimes resolves it, and sometimes doesn't.

**Comment by jpieper** [comment]
A reported recipe is to put 2 servos on bus 3, then start tview with `sudo tview --pi3hat-cfg '3=3,4' -t3-4`

**Comment by jpieper** [comment]
I've now tested locally with two servos, id 3 and 4, on bus 3 of a pi3hat and was unable to make it fail.  This was with a pi3hat r4.4b, and packages:

```
moteus 0.3.25
moteus-gui 0.3.25
moteus-pi3hat 0.3.12
```

I started up tview, it enumerated everything fine.  Then I expanded multiple channels on each servo, plotting some random values, and everything kept working.  I did this 6 or 7 times with no issues, across 3 power cycles.

To make progress here, I'll probably need to provide the reporter with instrumented packages that collect debugging information.
